examples/spark2/src/main/scala/org/apache/carbondata/examples/utils/StreamingCleanupUtil.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.examples.utils
+
+import java.io.IOException
+
+import scala.tools.nsc.io.Path
+
+// scalastyle:off println
+object StreamingCleanupUtil {
```

Comment:
I think it is better to move the CSV data generation to this utility object, and change its name to `StreamingExampleUtil`

Reply:
I agree, csv data generation can be part of utility object. I'll change it.